src/components/ErrorPage.tsx

```diff
@@ -1,59 +1,75 @@
-import { Box, Button, Center, Flex, Heading, Link } from "@chakra-ui/react";
+import { Box, Button, Center, Link, VStack } from "@chakra-ui/react";
+
+import { useOffboardingModal } from "./Offboarding/useOffboardingModal";
+import { ModalContentWrapper } from "./Onboarding/ModalContentWrapper";
+import { downloadBackupFile } from "./Onboarding/restoreBackupFile/utils";
+import { NoticeIcon, ReloadIcon } from "../assets/icons";
+import BackgroundImage from "../assets/onboarding/background_image.png";
+import colors from "../style/colors";
 
 const feedbackEmailBodyTemplate =
   "What is it about? (if a bug report please consider including your account address) %0A PLEASE FILL %0A%0A What is the feedback? %0A PLEASE FILL";
 
-// TODO: prepare for Beta release properly
 export const ErrorPage: React.FC = () => {
+  const { modalElement: OffboardingModal, onOpen: onOpenOffboardingModal } = useOffboardingModal();
+
   const onRefresh = () => {
     window.location.reload();
   };
 
-  const onBackup = () => {
-    const jsonString = `data:text/json;chatset=utf-8,${encodeURIComponent(
-      JSON.stringify(window.localStorage)
-    )}`;
+  return (
+    <Center height="100vh" padding="60px" backgroundImage={BackgroundImage} backgroundSize="cover">
+      <Box
+        width="100%"
+        maxWidth="460px"
+        height="100%"
+        maxHeight="480px"
+        padding="40px"
+        background={colors.gray[900]}
+        border="1px"
+        borderColor={colors.gray[700]}
+        borderRadius="8px"
+        boxShadow="0px 0px 30px rgba(0, 0, 0, 0.30)"
+      >
+        <ModalContentWrapper
+          icon={<NoticeIcon />}
+          subtitle="Please refresh the app or use one of the following options:"
+          title="Oops! Something went wrong!"
+        >
+          <VStack width="100%" spacing="16px">
+            <Button width="100%" onClick={downloadBackupFile} size="lg">
+              Save Backup
+            </Button>
 
-    const link = document.createElement("a");
-    link.href = jsonString;
-    link.download = "data.json";
+            <Button width="100%" onClick={onOpenOffboardingModal} size="lg" variant="warning">
+              Off-board Wallet
+            </Button>
 
-    link.click();
-  };
+            <Button width="100%" size="lg" variant="tertiary">
+              <Link
```

Comment:
please add `_hover={{textDecoration: "none"}}` so that it's not underlined on hover